samples/UrlHelperSample/project.json

```diff
@@ -1,12 +1,11 @@
 {
     "commands": {
-        "web": "Microsoft.AspNet.Hosting server=Microsoft.AspNet.Server.WebListener server.urls=http://localhost:5001",
-        "kestrel": "Microsoft.AspNet.Hosting --server Microsoft.AspNet.Server.Kestrel --server.urls http://localhost:5000"
+        "weblistener": "Microsoft.AspNet.Hosting server=Microsoft.AspNet.Server.WebListener server.urls=http://localhost:5001",
```

Comment:
Don't see a reason to stick w/ port 5001. Why isn't this just `Microsoft.AspNet.Server.WebListener`?